app/controllers/downgrade_controller.rb

```diff
@@ -1,17 +1,17 @@
 class DowngradeController < ApplicationController
   def new
   end
-
+  # Change user role to standard then...
   def create
-    current_user.update_attribute(:role, 0)
-    @wikis = Wiki.all
+    current_user.standard?
+    @wikis = current_user.wikis
     @wikis.each do |wiki|
-      if wiki.user.id == current_user.id
+      # ... make private wikis to public.
         wiki.update_attribute(:private, false)
       end
```

Comment:
Does this work? In this change, it looks like you have an extra end here that wasn't deleted when you got rid of your if statement